pyproject.toml

```diff
@@ -0,0 +1,81 @@
+[tool.poetry]
+name = "python-can"
+version = "3.2.0"
+description="Controller Area Network interface module for Python"
+license = "LGPL-3.0-only"
+authors = ["Brian Thorne <brian@thorne.link>"]
```

Comment:
It wouldn't be controversial to remove the `authors` and perhaps just list the mailing address email `python-can@googlegroups.com` under maintainers